README.md

```diff
@@ -1,82 +1,52 @@
 ![logo](https://raw.githubusercontent.com/vmware-tanzu/carvel/master/logos/CarvelLogo.png)
 
 # Carvel
-
+ 
 Carvel provides a set of reliable, single-purpose, composable tools that aid in your application building, configuration, and deployment to Kubernetes.
-
+ 
+Carvel is a set of tools designed to ease lifecycle management of your Kubernetes workloads. The origin of Carvel begins with Dmitriy Kalinin and Nima Kaviani not being satisfied with existing tools to deploy Kubernetes workloads. These tools were monolithic, error-prone, and hard to debug. Carvel promises a better way, one that extracts common app configuration into a library for use by all your applications.
```

Comment:
This might be a bit nitpicky but writing/editing is difficult. The start of this sentence feels repetitive of the line above. Perhaps we change "Carvel is a set of tools designed to ease..." to "Carvel eases...".